collect_app/src/main/java/org/odk/collect/android/formmanagement/ServerFormDownloader.java

```diff
@@ -210,18 +215,22 @@ private boolean isSubmissionOk(Map<String, String> parsedFields) {
 
         boolean installEverything(String tempMediaPath, FileResult fileResult, Map<String, String> parsedFields, String formsDirPath) {
             UriResult uriResult = null;
+
             try {
-                uriResult = findExistingOrCreateNewUri(fileResult.file, parsedFields, formsDirPath);
-                if (uriResult != null) {
-                    // move the media files in the media folder
-                    if (tempMediaPath != null) {
-                        File formMediaPath = new File(uriResult.getMediaPath());
-                        FileUtils.moveMediaFiles(tempMediaPath, formMediaPath);
-                    }
-                    return true;
-                } else {
-                    Timber.w("Form uri = null");
+                // Copy form to forms dir
+                File formFile = new File(formsDirPath, fileResult.file.getName());
```

Comment:
We now need to copy the file at the end. This was missing before, but we got away with it because the form was created in the final destination instead of the temp directory.